AppController/djinn.rb

```diff
@@ -3153,8 +3071,11 @@ def backup_appcontroller_state()
     APPS_LOCK.synchronize {
       local_state = {'@@secret' => @@secret }
       DEPLOYMENT_STATE.each { |var|
+        v = nil
```

Comment:
I suggest we abstain from single letter var names. It's harder to read and maintain.